docs/rfc/active/business_glossary/README.md

```diff
@@ -0,0 +1,358 @@
+
+## Metadata Model Enhancements 
+
+There will be 1 top level GMA [entities](../../../what/entity.md) in the design: businessTerm (Business Glossary).
+It's important to make businessTerm as a top level entity because it can exist without a Dataset and can be defined independently by the business team.
+
+### URN Representation
+We'll define a [URNs](../../../what/urn.md): `BusinessTermUrn`.
+These URNs should allow for unique identification of business term.  
+
+A business term  URN (BusinessTermUrn) will look like below:
+```
+urn:li:businessTerm:(<<namespace>>,<<name>>)
+```
+
+A Dataset Field URN(DatasetFieldUrn will be like below (this is being added as part of another RFC)
+```
+urn:li:datasetField:(<datasetUrn>,<fieldPath>)
+```
+
+### New Snapshot Object
+There will be new snapshot object to onboard business terms along with definitions
+
+Path : metadata-models/src/main/pegasus/com/linkedin/metadata/snapshot/
+```java
+/**
+ * A metadata snapshot for a specific BusinessTerm entity.
+ */
+record BusinessTermSnapshot {
+
+  /**
+   * URN for the entity the metadata snapshot is associated with.
+   */
+  urn: BusinessTermUrn
+
+  /**
+   * The list of metadata aspects associated with the dataset. Depending on the use case, this can either be all, or a selection, of supported aspects.
+   */
+  aspects: array[BusinessTermAspect]
+}
+```
+
+Path : metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/
+
+### BusinessTermAspect
+There will be new aspect defined to capture the required attributes & ownership information
+
+```
+/**
+ * A union of all supported metadata aspects for a BusinessTerm
+ */
+typeref BusinessTermAspect = union[
+  BusinessTermProperties,
+  Ownership
+]
+```
+
+Business Term Entity Definition
+```java
+/**
+ * Data model for a Business Term entity
+ */
+record BusinessTermEntity includes BaseEntity {
+
+  /**
+   * Urn for the dataset
+   */
+  urn: BusinessTermUrn
+
+  /**
+   * Business Term native name e.g. CashInstrument
+   */
+  name: optional string
+
+}
+```
+
+### Entity BusinessTermInfo
+
+```java
+/**
+ * Properties associated with a BusinessTerm
+ */
+record BusinessTermInfo {
+
+  /**
+   * Definition of business term
+   */
+  definition: string
+
+  /**
+   * Source of the Business Term (INTERNAL or EXTERNAL) with default value as INTERNAL
+   */
+  termSource: EnumType
+
+  /**
+   * External Reference to the business-term (URL)
+   */
+  sourceRef: optional string
+
+ /**
+   * URI of the external source when the term is borrowed from external
+   * e.g: fibo-fbc-fi-fi:CashInstrument
+   */
+  sourceURI: optional string
+
+  /**
+   * The abstracted URI such as https://spec.edmcouncil.org/fibo/ontology/FBC/FinancialInstruments/FinancialInstruments/CashInstrument.
+   */
+  uri: optional Uri
```

Comment:
Can you explain how uri And sourceURI are different .. how should the producers fill these fields and how should readers interpret these fields as ?

Reply:
Let me give an example here from FIBO
sourceURI : fibo-fnd-dt-fd:DateTime
uri : https://spec.edmcouncil.org/fibo/ontology/FND/DatesAndTimes/FinancialDates/DateTime
I see sourceURI will be redundant, one can always refer the sourceURI by navigating to the link (uri). Will keep only uri, with attribute name as sourceURL

Reply:
updated the respective section